Extended Documentation

koepalex opened the pull request:
Added description on how to integrate new information model into OPC Server with DataTypes server as example

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/OPCFoundation/UA-.NETStandard?pullRequest=938) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/OPCFoundation/UA-.NETStandard?pullRequest=938) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/OPCFoundation/UA-.NETStandard?pullRequest=938) it.</sub>

AlinMoldovean commented on SampleApplications/Workshop/DataTypes/README.md:
```diff
@@ -0,0 +1,7 @@
+﻿## DataTypes Server
+
+This server is build using the UA-.NETStandard stack as example, how to use data types.
```
small adding: 
This server is build using the UA-.NETStandard stack as example of how to define and use custom data types.

koepalex replied:
updated the text

mregen commented on SampleApplications/Workshop/DataTypes/_docs/IntegrateOwnInformationModel.md:
```diff
@@ -0,0 +1,79 @@
+   1. Create a sub-folder named "Output"
+2. Copy the model itself into this folder e.g. MyInformationModel.xml into UA-.NETStandard/SampleApplications/Workshop/DataTypes/Common/MyInformationModel
+3. Modify [BuildDesign.bat](./Common/BuildDesign.bat) and add the following lines
+
```
one small nitpick, since adding a model to a library is rather uncommon I'd describe the process as if the new model is added to the actual server. Hopefully just some path changes ..

koepalex replied:
I really liked the design of DataTypes server, to add the model into a library, so it will be possible use the library on client aswell on server (and for testing). 

if you say it is uncommon I can update the documentation - should I copy the BuildDesign.bat also to Server or link it from the current location?

koepalex commented:
> Hi Alexander,
> 
> Thanks for this update!
> 
> I think is better to put the whole text in README.md as we have it for [GDS](https://github.com/OPCFoundation/UA-.NETStandard/blob/master/SampleApplications/Samples/GDS/README.md)

I copied the content into README.md, deleted the _docs folder and updated the links

AlinMoldovean commented:
> 
> 
> > Hi Alexander,
> > Thanks for this update!
> > I think is better to put the whole text in README.md as we have it for [GDS](https://github.com/OPCFoundation/UA-.NETStandard/blob/master/SampleApplications/Samples/GDS/README.md)
> 
> I copied the content into README.md, deleted the _docs folder and updated the links

Please make sure to push the changes.
They are not yet in the PR

koepalex commented:
Updated with latested information:
* removed unnecessary loading of schema files from documentation and DataTypesNodeManager

koepalex commented:
Is there a another way than new commit to trigger checks? The build error seem not to be related with my changes

AlinMoldovean commented:
> 
> 
> Is there a another way than new commit to trigger checks? The build error seem not to be related with my changes

The failed tests have the "Re-run" option

![image](https://user-images.githubusercontent.com/13765819/79118059-6ff2be80-7d95-11ea-94fe-6a41e4e5e8f3.png)
